modules/vaos/app/services/vaos/configuration.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module VAOS
+  class Configuration < Common::Client::Configuration::REST
+    def base_path
+      Settings.va_mobile.url
+    end
+
+    def service_name
+      'VAOS'
+    end
+
+    def connection
+      Faraday.new(base_path, headers: base_request_headers, request: request_options) do |faraday|
+        faraday.use :breakers
+        faraday.use Faraday::Response::RaiseError
+
+        faraday.request :json
+
+        faraday.response :betamocks if mock_enabled?
+        faraday.response :snakecase
+        faraday.response :json, content_type: /\bjson$/
+        faraday.adapter Faraday.default_adapter
+      end
+    end
+
+    def mock_enabled?
+      [true, 'true'].include?(Settings.va_mobile.mock)
```

Comment:
Everywhere else we use a more idiomatic pattern `Settings.va_mobile.mock || false` or something similar

Reply:
did it this to only allow the boolean true or the string 'true', e.g. "fals" is truthy. It's also in the generator for modules, although I have to admit I'm biased because I wrote it 😄 
https://github.com/department-of-veterans-affairs/vets-api/blob/71b87a85784cc236710717dc823d8b6ab2266a28/lib/generators/module/templates/app/services/configuration.rb.erb#L27

Reply:
I think I would agree with Alastair here that this is technically more safe. Since any value other than `false` or `nil` would evaluate as a truthy. 

Might want to make changes elsewhere that are doing `Settings.va_mobile.mock || false`